ddtrace/contrib/responder/patch.py

```diff
@@ -0,0 +1,109 @@
+
+# stdlib
+
+# project
+import ddtrace
+from ddtrace.ext import http
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+from ddtrace.utils.wrappers import unwrap as _u
+from ddtrace.propagation.http import extract_context_from_http_headers
+
+# 3p
+import responder
+patches = []  # will be overriden when patched funcs are defined.
+
+
+def patch():
+    if _is_patched(responder):
+        return
+    _set_patched(responder, True)
+
+    for obj, mname, fname, func in patches:
+        _w(mname, fname, func)
+
+
+def unpatch():
+    if not _is_patched(responder):
+        return
+    _set_patched(responder, False)
+
+    for obj, _, prop, _ in patches:
+        if '.' in prop:
+            attr, _, prop = prop.partition('.')
+            obj = getattr(obj, attr, object())
+        _u(obj, prop)
+
+
+async def _api_call(wrapped, instance, args, kwargs):
```

Comment:
Pull these wrapper functions into a `wrappers.py`.

Reply:
why? just convention?

Reply:
a general convention that we don't enforce - i'm inclined to do separate the wrapped functions from the patching code, but might be overkill for simpler integrations.

Reply:
i think i'll leave, relatively simple.